opm/simulators/utils/DamarisOutputModule.cpp

```diff
@@ -133,6 +133,7 @@ setupDamarisWritingPars(Parallel::Communication comm, const int n_elements_local
     int64_t temp_int64_t[1];
     temp_int64_t[0] = static_cast<int64_t>(elements_rank_offsets[rank]);
     damaris_err = damaris_set_position("PRESSURE", temp_int64_t);
+    damaris_err = damaris_set_position("GLOBAL_CELL_INDEX", temp_int64_t);
```

Comment:
Since I don't know anything about Damaris except that this looks like a return-code based interface, I'm going to ask a probably naive question.  Is it safe to make multiple calls to `damaris_set_position()` without checking `damaris_err` between each call?  Suppose the `"PRESSURE"` call fails in some way and returns an error code, will the subsequent `"GLOBAL_CELL_INDEX"` call potentially succeed and overwrite the error code thus losing information that there was an error for `"PRESSURE"`?

Reply:
Thanks, I will amend.